packages/desktop/src/rtl-tests/channel.test.tsx

```diff
@@ -563,6 +563,85 @@ describe('Channel', () => {
     `)
   })
 
+  it.skip('user can enter a multi-line message and then traverse it with arrow keys', async () => {
+    const { store, runSaga } = await prepareStore(
+      {},
+      socket // Fork state manager's sagas
```

Comment:
I wonder if we really need to fork sagas here... it'd probably be better to isolate input component for this test. On the other hand it may be useful to check whether keyboard mapping works in wide context - can we achieve it without having to populate the store?

Reply:
@siepra I've removed this fork and only render the component in the test, but unfortunately it still fails.